2Dgame/Assets/Scripts/PlatformSpawnerScript.cs

```diff
@@ -56,15 +56,70 @@ public class PlatformSpawnerScript : MonoBehaviour {
     GameObject PlatformScheme(int percentNormal, int percentTrampoline, int percentMoving)
```

Comment:
Could these two methods perhaps be condensed to one? Or maybe this one makes a call like PlatformScheme(a,b,c, 0) to the other function.